app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError(
+                "NotVaccinatedError should be raised with a message")
```

Comment:
it seems to me that this is not descriptive message for user, the same is for other exceptions